src/test/README.md

```diff
@@ -0,0 +1,30 @@
+
+# Unit Tests
+
+## Running Tests
+
+Unit tests are bundled in the `rippled` executable and can be executed using the
+`--unittest` parameter. Without any arguments to this option, all non-manual
+unit tests will be executed. If you want to run one or more manual tests, you
+must specify it by suite or full-name (e.g. `ripple.app.NoRippleCheckLimits` or
+just `NoRippleCheckLimits`).
+
+More than one suite or group of suites can be specified as a comma separated
+list via the argument. For example, `--unittest=beast,OversizeMeta` will run
```

Comment:
I'm noting that the examples in this file use `=` with no spaces.  But the `--help` says to use spaces in the arguments.  I tried both.  They both work.  It might be good, however, for our messaging to be consistent.

My personal preference is to use an `=` as the first argument separator and commas (with no spaces) as the remaining separators.  Just like you've shown here.  That visually glues all of the options together.  I haven't really paid any attention to boost::program_options.  I bet it would be hard to get it to show `=` instead of spaces.  Bummer.  It's not worth a lot of messing around.